website/docs/r/cognitive_deployment.html.markdown

```diff
@@ -78,7 +78,7 @@ A `scale` block supports the following:
 
 * `family` - (Optional) If the service has different generations of hardware, for the same SKU, then that can be captured here. Changing this forces a new resource to be created.
 
-* `capacity` - (Optional) Tokens-per-Minute (TPM). If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted. Default value is `1`. Changing this forces a new resource to be created.
+* `capacity` - (Optional) Tokens-per-Minute (TPM). Unit of this value is thousands of Token-per-Minute. Default value is `1` which indicates limitation is `1000` tokens per minute. If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted. Changing this forces a new resource to be created.
```

Comment:
That is way better, but is this maybe more clear? Also should we add a link to the official product documentation explaining what this field does exactly? 

```suggestion
* `capacity` - (Optional) Tokens-per-Minute (TPM). The unit of measure for this field is in the thousands of Tokens-per-Minute. Defaults to `1` which means that the limitation is `1000` tokens per minute. If the resources SKU supports scale in/out then the capacity field should be included in the resources configuration. If the scale in/out is not supported by the resources SKU then this field can be safely omitted. Changing this field forces a new resource to be created.
```

Reply:
Great, and this is the link for TPM https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/quota?tabs=rest